core/client/fs/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -251,29 +249,21 @@ public BlockInStream getInStream(BlockInfo info, InStreamOptions options,
    */
   public BlockOutStream getOutStream(long blockId, long blockSize, WorkerNetAddress address,
       OutStreamOptions options) throws IOException {
-    if (blockSize == -1) {
```

Comment:
I checked the src code, blockSize will never be -1 after this PR.
using blockSize as a switch to indicate existence is anyway a bad practise.